lib/src/widget/button.dart

```diff
@@ -177,7 +177,7 @@ class _NeumorphicButtonState extends State<NeumorphicButton> {
 
   Widget _build(BuildContext context) {
     final appBarPresent = NeumorphicAppBarTheme.of(context) != null;
-    final appBarTheme = NeumorphicTheme.of(context).current.appBarTheme;
+    final appBarTheme = NeumorphicTheme.of(context)?.current?.appBarTheme;
```

Comment:
I'd suggest doing this instead:
```
final appBarTheme = NeumorphicTheme.currentTheme(context).appBarTheme;
```
Reason being that the `NeumorphicTheme.of(context)?.current?.appBarTheme;` still has a possibility of returning null (which will still throw a null reference exception). `NeumorphicTheme.currentTheme` is guaranteed to not return null.

Reply:
okay

Reply:
I have done the required changes @josiahsrc